4_advanced_usage/77_distributed_rudder.txt

```diff
@@ -73,6 +73,111 @@ cd rudder-multiserver-setup
 
 ----
 
+==== Install rudder-db manually
```

Comment:
I actually think this should be a separate section, just before "distributed_rudder", so that it's visible in the index. And in some ways, this is a "light" version of a fully-distributed server install.